cmd/applications-rp/main.go

```diff
@@ -43,6 +43,7 @@ import (
 	"github.com/radius-project/radius/pkg/ucp/ucplog"
 
 	corerp_setup "github.com/radius-project/radius/pkg/corerp/setup"
+	dsrp_setup "github.com/radius-project/radius/pkg/datastoresrp/setup"
```

Comment:
Not sure about the underscore in the alias naming. We should maybe check the GO conventions.

Reply:
I didn't see anything specific to import naming conventions. This is how the corerp imports are defined so I'm going to keep it for consistency but we can revisit this if we decide we need to change the format